docs/migration-tm2-to-tm3.md

```diff
@@ -0,0 +1,51 @@
+# Migrate Tasking Manager from version 2 to version 3
+
+Migrating from TM2 to TM3 consists of two main parts: 1) installing the new TM3 application and setting up its unpopulated database and 2) migrating the data from the original TM2 database to the new TM3 database.
+
+## Installation
+
+The method by which you install TM3 on your own computer will vary, but you should be in the following position to continue on with the migration:
+
+* You have downloaded a local copy of the TM3 repository on your computer.
+* You have [installed python3.6 (preferably in a virtual environment) and all of the requirements via pip](https://github.com/hotosm/tasking-manager#dependencies).
+* You have [installed all of the nodeJS modules in the client folder and gulp built the client](https://github.com/hotosm/tasking-manager#client-development).
+* You have [set all the environmental variables needed to run the TM3](https://github.com/hotosm/tasking-manager#environment-vars).
+* You have [created a new database for TM3](https://github.com/hotosm/tasking-manager/wiki/Dev-Ops) with the same owner as your TM2 database.
+* You have [populated the database schema for TM3 using alembic](https://github.com/hotosm/tasking-manager#creating-the-db).
+
+A check to ensure all of these conditions are met is to start the TM3 app, launch the main homepage , and click on while watching the console. It will say there are currently no projects, which is expected since you have a skeleton database, but you should make sure no errors are thrown when the application tries checking for projects. If the page loads successfully and no errors occur, then you should be set!
+
+### Installation Example Walkthroughs
```

Comment:
There are some links to the wiki in that section. Should we move the linked content to the `docs` folder or just link to the `setup-development.md` file?

Reply:
Right. I put a link to the dev environment setup and rephrased the following sentence a bit.